sinks/channel/channel_sink.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package channel
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/hashicorp/eventlogger"
+)
+
+// ChannelSink is a sink node which sends
+// the event to a channel
+type ChannelSink struct {
+	mu sync.Mutex
+
+	eventChan chan *eventlogger.Event
+}
+
+var _ eventlogger.Node = &ChannelSink{}
+
+// newChannelSink creates a ChannelSink
+func NewChannelSink(c chan *eventlogger.Event) (*ChannelSink, error) {
```

Comment:
Ditto type the direction of the channel.

Reply:
See my prev. comment on timeout duration.